process/Issue_Triaging.md

```diff
@@ -0,0 +1,113 @@
+
+Triaging an issue is a multi-step process that is collaboratively performed by community. Triaging an issue usually takes around one business day but may take longer, for example, when the reviewer is not around. Goal of triaging is to provide you with a clear understanding of what will happen to your issue. For example, after your feature request was triaged you know whether we plan to tackle the issue or whether we'll wait to hear what the broader community thinks about this request.
+
+
+## Terminologies
+
+* TSC: Technical Steering Committee. An informal group of people who meet on a monthly basis and discuss technical topics related to software development. Everyone is welcome to join the Technical Steering Committee. Refer to the charter for more details.
+
+
+## Process
+
+The basic process of triaging issues is
+
+|First Step     |Next Step            |Result     |
+|---            |---                  |---        |
+|TSC review ->  |Accepted, or         |
+|               |Community review ->  |Closed, or |
+|               |                     |Accepted   |
+
+Please note:
+1. TSC reviewers can accept an issue but cannot close an issue before it is being reviewed by the community (the "Community Review" step).
+2. We are much less likely to accept PRs that have no associated accepted issues
+
+Issues can be at four major states during this process. They are easily identifiable:
+
+|State|What your GitHub issue looks like|
+|---|---|
+|TSC Review|matches the query `label:"tsc-review"`|
+|Community Review|matches the query `milestone:"Backlog Candidates"`|
+|Accepted|has any milestone except `Backlog Candidates`|
+|Closed|matches the query `is:closed`|
+
+In the rest of this document, we'll go into more detail about each of the activities of triaging.
+
+
+## TSC Review
+
+TSC rotates a primary and a secondary reviewer every week. The reviewers triage issues by assigning the following labels:
+
+### Categorization Labels
+
+Each issue must have a **type** label. Most type labels are grey, some are yellow. Bugs are grey with a touch of red.
+
+|Type|Description|
+|---|---|
+|[`needs-more-info`](https://github.com/maplibre/maplibre-gl-native/labels/xx) | not possible to assign a type label due to missing information|
+|[`bug`](https://github.com/maplibre/maplibre-gl-native/labels/xx) | the implementation of a feature is not correct|
+|[`feature-request`](https://github.com/maplibre/maplibre-gl-native/labels/xx) | request for a new feature|
+|[`tech-debt`](https://github.com/maplibre/maplibre-gl-native/labels/xx) | improve the implementation/architecture|
+|[`engineering`](https://github.com/maplibre/maplibre-gl-native/labels/xx) | issues related to our engineering system or our processes|
+|[`polish`](https://github.com/maplibre/maplibre-gl-native/labels/xx) | a feature could be improved, but not necessarily a bug|
+|[`*question`](https://github.com/maplibre/maplibre-gl-native/labels/xx) | we should direct questions to StackOverflow|
```

Comment:
We also have discussions available.

Do we need to triage also discussions? 😊

Reply:
No. An issue that is a question should be converted to discussion, IMHO.